txpool/txpool_grpc_server.go

```diff
@@ -51,7 +51,7 @@ var TxPoolAPIVersion = &types2.VersionReply{Major: 1, Minor: 0, Patch: 0}
 type txPool interface {
 	ValidateSerializedTxn(serializedTxn []byte) error
 
-	Best(n uint16, txs *types.TxsRlp, tx kv.Tx, onTopOf, availableGas uint64, isMining bool) (bool, error)
+	PeekBest(n uint16, txs *types.TxsRlp, tx kv.Tx, onTopOf, availableGas uint64) (bool, error)
```

Comment:
Don't we need to add `YieldBest` to the interface?

Reply:
We're only using PeekBest there as we're getting everything from the txpool.  The interface is local to the grpc server module